docs/plugins/repository-s3.asciidoc

```diff
@@ -63,7 +59,7 @@ bin/elasticsearch-keystore add s3.client.default.secret_key
 ----
 
 The following are the available client settings. Those that must be stored in the keystore
-are marked as `Secure`.
+are marked as `Secure`; all other settings are public and may be included in the `elasticsearch.yml` file.
```

Comment:
Per above, I'd just say "...all other settings can be included in the `elasticsearch.yml` file".